merlin/schema/tags.py

```diff
@@ -26,30 +27,45 @@ class Tags(Enum):
     LIST = "list"
     SEQUENCE = "sequence"
     TEXT = "text"
-    TEXT_TOKENIZED = "text_tokenized"
+    TOKENIZED = "tokenized"
     TIME = "time"
 
     # Feature context
+    ID = "id"
     USER = "user"
-    USER_ID = "user_id"
     ITEM = "item"
-    ITEM_ID = "item_id"
     SESSION = "session"
-    SESSION_ID = "session_id"
     CONTEXT = "context"
 
     # Target related
     TARGET = "target"
-    BINARY_CLASSIFICATION = "binary_classification"
-    MULTI_CLASS_CLASSIFICATION = "multi_class"
     REGRESSION = "regression"
+    CLASSIFICATION = "classification"
+
+    BINARY = "binary"
```

Comment:
I am wondering if this (& `MULTI_CLASS`) could be infered automatically based on the int_domain?

Reply:
could be represented outside of tags and instead with a property or method

would there ever be a use-case where we'd want to know if a column is binary or multi-class before it's been converted to an int? e.g. pre Categorify op, perhaps these are represented as strings.

Reply:
It's possible that they could be inferred, but I think that's outside the scope of this PR (which is, after all, about making things more atomic) 😅